core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -2839,7 +2839,7 @@ SDDkwd__(ISO_MAPPING,           (char *)SQLCHARSETSTRING_ISO88591),
   DD_____(PCODE_DEBUG_LOGDIR,        ""  ), // Pathname of log directory for PCode work
   DDint__(PCODE_EXPR_CACHE_CMP_ONLY, "0" ), // PCode Expr Cache compare-only mode
   DDint__(PCODE_EXPR_CACHE_DEBUG,    "0" ), // PCode Expr Cache debug (set to 1 to enable dbg logging)
-  DDint__(PCODE_EXPR_CACHE_ENABLED,  "1" ), // PCode Expr Cache Enabled (set to 0 to disable the cache)
+  DDint__(PCODE_EXPR_CACHE_ENABLED,  "0" ), // PCode Expr Cache Enabled (set to 0 to disable the cache)
```

Comment:
On EXPECTED023, I was trying to change the text of an error message which incorrectly had the default schema as SEABASE instead of SCH. But I think that change had already been made (between the time I checked out and created this PR). Instead of my actual change, these DIFFs now show all the timestamp differences that are likely handled by the FILTER? I was not aware of the issue  in HSTable::makeSampleName. Thank you for explaining.